invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/administration_page.html

```diff
@@ -7,9 +7,9 @@
   under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- extends "invenio_theme/page.html" -%}
+{%- extends config.BASE_TEMPLATE -%}
```

Comment:
nit: I am not sure if we want to be more flexible like https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/templates/semantic-ui/invenio_requests/details/index.html#L10C1-L14C12